src/libs/OptionsListUtils.js

```diff
@@ -143,6 +142,27 @@ function getParticipantNames(personalDetailList) {
     return participantNames;
 }
 
+/**
+ * A very optimized method to remove unique items from an array.
+ * Taken from https://stackoverflow.com/a/9229821/9114791
+ *
+ * @param {Array} items
+ * @returns {Array}
+ */
+function uniqFast(items) {
+    const seenItems = {};
+    const result = [];
+    let j = 0;
+    for (let i = 0; i < items.length; i++) {
+        const item = items[i];
+        if (seenItems[item] !== 1) {
+            seenItems[item] = 1;
+            result[j++] = item;
```

Comment:
Wait it is slower to push an item into an array than do this? 🤯

Reply:
I think the general answer is "yes, it's faster", but there are differences between JS engines which make the testing somewhat unreliable. Generally, `push()` does some extra bits of processing from the base JS object that arrays extend from, whereas using an index approach bypassing that little bit of overhead.